adapters/trustedstack/trustedstack.go

```diff
@@ -0,0 +1,121 @@
+package trustedstack
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v2/adapters"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/errortypes"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+)
+
+type adapter struct {
+	endpoint string
+}
+
+func (a *adapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	var errs []error
+
+	reqJson, err := json.Marshal(request)
+	if err != nil {
+		errs = append(errs, err)
+		return nil, errs
+	}
+
+	headers := http.Header{}
+	headers.Add("Content-Type", "application/json;charset=utf-8")
+
+	return []*adapters.RequestData{{
+		Method:  "POST",
+		Uri:     a.endpoint,
+		Body:    reqJson,
+		Headers: headers,
+	}}, errs
+}
+
+func (a *adapter) MakeBids(internalRequest *openrtb2.BidRequest, externalRequest *adapters.RequestData, response *adapters.ResponseData) (*adapters.BidderResponse, []error) {
+	var errs []error
+
+	if response.StatusCode == http.StatusNoContent {
+		return nil, nil
+	}
+
+	if response.StatusCode == http.StatusBadRequest {
+		return nil, []error{&errortypes.BadInput{
+			Message: fmt.Sprintf("Unexpected status code: %d. Run with request.debug = 1 for more info", response.StatusCode),
+		}}
+	}
```

Comment:
consider using response util for status checks

https://github.com/prebid/prebid-server/blob/ec729e6e5d50a1306d76332fda52108b9d313c1d/adapters/response.go#L1-L28

Reply:
> Done.

@product-trustedstack  could you point out changes being done. File changes still has old code

Reply:
response.go only support "StatusBadRequest" and status not equals "StatusOk". For both of these error is returned from the response.go file. These two if statementshas been removed and response.go function "CheckResponseStatusCodeForErrors" is used not. However, in our case, we have "NoContent" Status code as well, where we are retuning both response and err as nil. This has been same, other two has been removed incorporating the suggested file above.